src/module/rolls/roll.ts

```diff
@@ -0,0 +1,322 @@
+// TODO:
+// - Convert to HTML for chat messages
+// - Rehydrate from chat message (will be needed for rerolls)
```

Comment:
i'm curious to see how you end up doing this! my first instinct would be to try putting some stringified JSON in a data attribute... but this version with `<script>` seems smarter, haha: https://stackoverflow.com/questions/8542746/store-json-object-in-data-attribute-in-html-jquery#answer-64154690

Reply:
Currently I'm embedding stringified JSON in a data-attribute, and that works just fine. I don't want to risk giving Foundry a script tag and having it stripped out.